src/validation.cpp

```diff
@@ -1304,6 +1374,7 @@ PackageMempoolAcceptResult MemPoolAccept::AcceptMultipleTransactions(const std::
         // updated if package replace-by-fee is allowed in the future.
         assert(!args.m_allow_replacement);
         m_viewmempool.PackageAddTransaction(ws.m_ptx);
+        ++num_prechecks_passed;
```

Comment:
why increment `num_prechecks_passed` here ?

Reply:
because otherwise it is always 0